### DISCUSSION TITLE
Introduction | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#887] [Q&A]
# Introduction | Seeed Studio Wiki

Get Started with SenseCAP Indicator Introduction

https://wiki.seeedstudio.com/SenseCAP_Indicator_How_To_Flash_The_Default_Firmware/

**Comment by pljda** [comment]
Hi,
I test the RP2040 part and dare say it is  very well explained (and do not forget to follows the instructions in the links). Thanks. The only "sad" point is that the "example file" to test and download from the Arduino IDE does not exist as such. 
Will test the other parts and hopefully will be able to integrate it to Home Assistant. 
Regarding HA, it seems that there is no simple route for its integration !
Philippe

**Comment by sebastianha** [comment]
Hi,

I do have a question: What is the exact difference between the T1000-A and T1000-E? According to the table above the first is for LoRa and the other one for Meshtastic.

Is it possible to change with firmware from an -E device to a -A device and vice versa?

Sebastian
